src/picongpu/include/particles/bremsstrahlung/SynchrotronFunctions.hpp

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright 2015 Heiko Burau
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "types.h"
+#include "cuSTL/container/HostBuffer.hpp"
+#include "cuSTL/container/HostBuffer.hpp"
+#include "cuSTL/cursor/Cursor.hpp"
+#include "cuSTL/cursor/navigator/PlusNavigator.hpp"
+#include "cuSTL/cursor/tools/LinearInterp1D.hpp"
+#include "cuSTL/cursor/BufferCursor.hpp"
+#include "simulation_defines.hpp"
+#include <boost/shared_ptr.hpp>
+#include <boost/array.hpp>
+#include <boost/numeric/odeint.hpp>
```

Comment:
new boost library: `Numeric.Odeint`, [header-only](https://gist.github.com/ax3l/7450482) ([ref](http://www.boost.org/doc/libs/1_59_0/libs/numeric/odeint/doc/html/boost_numeric_odeint/getting_started/usage__compilation__headers.html)).

- what version of boost do you require at minimum for this functionality to work?
- the [documentation]((http://www.boost.org/doc/libs/1_59_0/libs/numeric/odeint/doc/html/boost_numeric_odeint/getting_started/usage__compilation__headers.html)) states "It is also possible to include only parts of the library. This is the recommended way since it saves a lot of compilation time. " -> can you be more specific with your includes please?

can [trigger dependencies to](http://www.boost.org/doc/libs/1_59_0/libs/numeric/odeint/doc/html/boost_numeric_odeint/getting_started/overview.html) a couple of third party libs. does this affect you and do you require some? if yes, do we need to add additional dependencies in our environments?